tests/testtlswebsockets.nim

```diff
@@ -124,9 +124,7 @@ suite "Test websocket TLS transmission":
         check request.uri.path == "/wss"
         let ws = await createServer(request, "proto")
         let servRes = await ws.recv()
-
         check string.fromBytes(servRes) == testString
-        await ws.close()
```

Comment:
This isn't the proper fix, removing this close just hides the real issue, which seems to be related to the tls server in chronos, running it with the regular http server works as expected. 

@cheatfate can you please take a look, the closeWait hangs on windows on the tls server after both sockets are closed, roving the closing of one of the sockets seems to remove the hang. I traces it down to where all the processLoop features are cancelled and awaited, the hangs happens right there.

Reply:
Yes. This was a temporary hack. I assumed the issue was with WebSocket close. https://github.com/status-im/nim-ws/pull/9 Ported a couple of changes from this PR but the close didn't work. It could be an issue in the TLS server as you mentioned.

Reply:
@arijitAD, can you please create a minimal repro of this issue and _iff_ it still present, log it as a bug in chronos?

Reply:
Sure. I will do that.

Reply:
Turns out that the server callback needs to return a `dumbRespose()` otherwise it will attempt to write a response on an already closed connection. This is likely the cause of the hang on windows, but it's pretty terrible UX in general. We should add an accept call to chronos or to this implementation to avoid this types of issues.